frontend/components/DocumentFilters/DocumentFilters.jsx

```diff
@@ -37,13 +42,17 @@ export default class DocumentFilters extends Component {
 
     const {collection} = this.props
     const paramFilters = this.getFiltersFromParams()
-    this.state.filters = paramFilters
+
+    this.state = {
```

Comment:
@eduardoboucas it doesn't seem to. Confirm that this is picked up with you run `npm run test`?